build.cmd

```diff
@@ -204,6 +204,19 @@ call                                 "%__VSToolsRoot%\VsDevCmd.bat"
 
 REM =========================================================================================
 REM ===
+REM === Restore optimization profile data
+REM ===
+REM =========================================================================================
+
+echo %__MsgPrefix%Restoring the OptimizationData Package
```

Comment:
It would be great to add a command line option to build.cmd that would allow skipping this. In development scenarios, it is nice to be able to cut down the build time to the minimum possible and also enable build in disconnected scenarios.